test/test_models.py

```diff
@@ -178,7 +178,7 @@ def __init__(self):
             {
                 "a": 42,
                 "b": lambda a: math.pow(a, 5),
-                "counter": lambda: C(),
+                "counter": lambda: C,
```

Comment:
I suspect this line should be reverted so the constructor is invoked and `counter` gets an instance of `C` rather than the class `C`. Then on line 193 the parens should be removed, which previously were evaluating the lambda (and after this PR I believe are invoking the constructor).

Reply:
You're right, thanks for catching that! https://github.com/brunolange/artifax/commit/83c7483a175da446e3ea1f2cb962d3b342db0edd